QUEST_LV_0100.tsv

```diff
@@ -6403,101 +6403,101 @@ QUEST_LV_0100_20150317_006402	{memo X}Mr. Juan asked you to bring him Yukopus Le
+QUEST_LV_0100_20150323_006456	$Perhaps the source of evil in Kvailas Forest has become the forest itself? {nl} I only hope this is just an ill-founded worry.
+QUEST_LV_0100_20150323_006457	$Everything will be resolved when the evil force deep inside Kvailas Forest is dealt with. {nl}It's a pity that everyone is only after what's right in front of them.
+QUEST_LV_0100_20150323_006458	$Kvailas Forest is the worst. {nl}The evil force is unquestionably concentrated and even your lives are at stake. 
+QUEST_LV_0100_20150323_006459	$I miss my family in Cobalt Forest. {nl}And... I want to apologize to my brother for the fight we had.
+QUEST_LV_0100_20150323_006460	$I remember singing and soaking my feet in the stream waters with my older brother in Cobalt Forest.
+QUEST_LV_0100_20150323_006461	$Can you inform this to the elderly in Vieta Gorge for me? {nl}If you veer left and move up  from here, you'll quickly reach her. 
 QUEST_LV_0100_20150323_006462	$The most important root must be somewhere in Kvailas Forest. {nl}Maybe we've been reacting around too much. {nl}Cutting off the head root will reduce the forces in this forest.
-QUEST_LV_0100_20150323_006463	I knew it was the poison of Evonipons. {nl}Its the only poison this strong. 
+QUEST_LV_0100_20150323_006463	$I knew it was Evoniphon's poison. {nl}There is no one else that will use a poison this strong. 
 QUEST_LV_0100_20150323_006464	{memo X}Don't worry. I'll give you the recipe. {nl}Ridimed leaves from there will be enough to make it. 
-QUEST_LV_0100_20150323_006465	Is the antidote ready yet? {nl}That's terrible. He is getting worse.
-QUEST_LV_0100_20150323_006466	Evonipon sure are vicious. {nl}We need to hunt it down before it cause more damage. 
-QUEST_LV_0100_20150323_006467	Oh is it antidote? {nl}Thank you very much. I will let him take it right away. {nl}
-QUEST_LV_0100_20150323_006468	If it weren't for you, many workers could have lost their lives for nothing. {nl}I will report about this to Sir Uska. {nl}Truly.. You were sent by the Goddess!{nl}
-QUEST_LV_0100_20150323_006469	A monster in the Owl Grave is a fair bet for you. {nl}Isn't it? 
-QUEST_LV_0100_20150323_006470	{memo X}Chaparition of Escanciu Village took a lot of lives. {nl}Please defeat that monster in the name of the Goddess.
+QUEST_LV_0100_20150323_006465	$Is the antidote ready yet? {nl}The situation's grave. He is gradually getting worse.
+QUEST_LV_0100_20150323_006466	$Evoniphon sure employed a vicious poison. {nl}We need to hunt him down before he causes more damage... 
+QUEST_LV_0100_20150323_006467	$Oh, is this the antidote? {nl}Thank you very much. I will let him take it right away. {nl}
+QUEST_LV_0100_20150323_006468	$If it wasn't for you, many workers could have lost their lives for nothing. {nl}I will report about this to Sir Uska. {nl}Truly... You were sent by the Goddess!{nl}
```

Comment:
"If it wasn't for you, many workers could have lost their lives for nothing." -> "If it wasn't for you, many workers **would** have lost their lives for nothing."